langwatch/src/server/tracer/opentelemetry.ts

```
@@ -901,52 +901,67 @@ type RecursiveRecord = {
     }
-    return obj;
+
+    if (anyValuePair.boolValue != null) return anyValuePair.boolValue;
+    if (anyValuePair.intValue != null) return anyValuePair.intValue;
+    if (anyValuePair.doubleValue != null) return anyValuePair.doubleValue;
+    if (anyValuePair.bytesValue != null) return anyValuePair.bytesValue;
+
+    if (anyValuePair.kvlistValue)
+      return otelAttributesToNestedAttributes(anyValuePair.kvlistValue.values);
+
+    if (anyValuePair.arrayValue && anyValuePair.arrayValue.values)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)